### PR TITLE
Creating AWS Release Workflow

### PR DESCRIPTION
**Pull request by ian-flores**
This PR introduces a GitHub Action that streamlines the release process from a manual one into an automated one. Once a developer tags their commit with a tag in the `v*` format the Action will be executed. This includes both creating a GitHub release as well as updating the S3 bucket where the templates are offered to clients.

### CODE REVIEW

File: .github/workflows/release-cloudformation.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          files: |
+            aws/cloudformation/rstudio-standalone.yml
+    
+    - name: AWS Release
+      run: |
+        aws s3 cp aws/cloudformation/rstudio-standalone.yml \
+        s3://rstudio-cloud-tools/cloudformation/rstudio-standalone-${{github.ref_name}}.yml
+
+        aws s3 cp aws/cloudformation/rstudio-standalone.yml \
+        s3://rstudio-cloud-tools/cloudformation/rstudio-standalone.yml
```
Comment (colearendt) [inline .github/workflows/release-cloudformation.yml:50]: Missing a newline here

**Review by colearendt — COMMENTED** [review]
Interesting! Do you mind writing up a description that explains the process you are using / what other pieces to the access control exist? Or linking to a doc somewhere internally if there is a writeup elsewhere.

Probably worth adding someone on SRE to review this as well.

**Comment by ian-flores** [comment]
I worked on that documentation last week with @jspiewak from SRE. It's hosted here: https://app.tettra.co/teams/rstudio/pages/aws-how-to-use-roles-with-github-actions

File: .github/workflows/release-cloudformation.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
```
Comment (colearendt) [inline .github/workflows/release-cloudformation.yml:51]: I think this is a problem with your editor - the newline that's missing is the "end of file" newline

Comment (ian-flores) [reply .github/workflows/release-cloudformation.yml:51]: Fixed it now with adding it through the VSCode setting instead of manually

Comment (colearendt) [reply .github/workflows/release-cloudformation.yml:51]: Nice! It's still there though - you can see on the "Files" page - maybe you need to open / re-save the files?

Comment (ian-flores) [reply .github/workflows/release-cloudformation.yml]: Okay, now it has been addressed! 😄

**Review by colearendt — APPROVED** [review]
LGTM!

**Pull request merged**